src/stk500.c

```diff
@@ -1294,6 +1345,19 @@ static int stk500_set_sck_period(const PROGRAMMER *pgm, double v) {
 }
 
 
+static int stk500_get_sck_period(const PROGRAMMER *pgm, double *v) {
+  unsigned dur;
+  int rv = 0;
+
+  if ((rv = stk500_getparm(pgm, Parm_STK_SCK_DURATION, &dur))< 0) {
```

Comment:
Is the spacing around binary `<` deliberate? It's a bit unusual to have asymmetric spacing.

Reply:
Typo when staring to long at the screen during nighttime (same as above `return - 1;`). This is the reason that I do an automatic `clang-format -i ...` before I check in my own projects. And my `qtcreator` is told to do it automatically when saving.